thumbor/src/thumbor_/__main__.py

```diff
@@ -1,12 +1,34 @@
+import asyncio
+import sys
 from argparse import ArgumentParser
 
-from . import __version__
+from thumbor_.server import main as server_main
+from thumbor_.storage_manager import StorageManager
+
+
+async def manage_storage(delete_old=False):
```

Comment:
Is this ever called without `delete_old=True`?

Reply:
No, in this case should I remove the delete_old and just call the function directly to list and delete from the bucket if  delete expired is passed to the CLI?

Reply:
Done!